hawkbit-runtime/hawkbit-update-server/src/main/java/org/eclipse/hawkbit/app/aop/StrictHttpFirewallAdvice.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.app.aop;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.eclipse.hawkbit.exception.SpServerError;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.security.web.firewall.RequestRejectedException;
+import org.springframework.security.web.firewall.StrictHttpFirewall;
+import org.springframework.security.web.FilterChainProxy;
+import org.springframework.stereotype.Component;
+
+/**
+ * Advice class for the {@link FilterChainProxy} to handle the
```

Comment:
May I ask you to extend the comment and explain why you're using an advice instead of configuring the bean via:

```
@Bean
public HttpFirewall myFirewall() {
    // ...
}
```